packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.jsx

```diff
@@ -48,7 +86,25 @@ class VerticalViewLayoutBase extends React.PureComponent {
   setScrollingStrategy() {
     const { setScrollingStrategy } = this.props;
 
-    setScrollingStrategy(scrollingStrategy(this.layout.current, this.layoutHeader.current));
+    setScrollingStrategy(scrollingStrategy(
+      this.layout.current, this.layoutHeader.current, this.timeScale.current,
+    ));
+  }
+
+  setBorders(event) {
+    const { isLeftBorderSet, isTopBorderSet } = this.state;
+    if ((!event.target.scrollLeft && isLeftBorderSet)
+      || (event.target.scrollLeft && !isLeftBorderSet)) {
```

Comment:
It can be done with XOR operator - `event.target.scrollLeft ^ isLeftBorderSet`

Reply:
Lint disallows bitwise operators, including XOR. Should we use it anyway?

Reply:
Yes, just ignore the rule for this line

Reply:
XOR does not work with numbers (and `scrollLeft` is number) and booleans as we want in this case. For instance, `300 ^ true` is 301. Which is, in current context, will result in `true`, but we need false here.

Reply:
Then `!!event.target.scrollLeft ^ isLeftBorderSet`